<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-ABLBB1N" failures="1" tests="3" name="Test" time="9.75" errors="0" timestamp="2024-11-14T13:32:21 IST">
  <testcase classname="day46.TestCase" name="testLogo" time="0.05"/>
  <testcase classname="day46.TestCase" name="testAppUrl" time="0.01">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at day46.TestCase.testAppUrl(TestCase.java:46)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testAppUrl -->
  <testcase classname="day46.TestCase" name="testHomePageTitle" time="0.0">
    <skipped/>
  </testcase> <!-- testHomePageTitle -->
</testsuite> <!-- Test -->
